packages/camera/camera_windows/pigeons/messages.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(PigeonOptions(
+  dartOut: 'lib/src/messages.g.dart',
+  cppOptions: CppOptions(namespace: 'camera_windows'),
+  cppHeaderOut: 'windows/messages.g.h',
+  cppSourceOut: 'windows/messages.g.cpp',
+  copyrightHeader: 'pigeons/copyright.txt',
+))
+
+/// Pigeon version of pltaform interface's ResolutionPreset.
+enum PlatformResolutionPreset { low, medium, high, veryHigh, ultraHigh, max }
+
+// Pigeon version of to MediaSettings.
```

Comment:
Pigeon version of MediaSettings?

Should this be a doc comment? Pretty minimal either way so probably doesn't matter much, but the others appear to be doc comments.

Reply:
Fixed and fixed!